front/lib/api/assistant/actions/websearch.ts

```diff
@@ -102,9 +100,9 @@ export class WebsearchConfigurationServerRunner extends BaseActionConfigurationS
     }
 
     return new Ok({
-      name: name ?? "web_search",
+      name: name,
       description:
-        description ?? "Perform a web search and return the top results.",
+        description || "Perform a web search and return the top results.",
```

Comment:
nit: keep ?? and name,

Reply:
I think || is safer will give us a chance to get a description if we don't have one